lib/pr_checker.js

```diff
@@ -211,6 +252,36 @@ class PRChecker {
     // 3. is not authored by the people opening the PR
     return true;
   }
+
+  checkCommitsAfterReview(commit, review, logger) {
+    const commitIndex = commit.length - 1;
```

Comment:
With this we will need to guarantee that no one alters the order of the arrays. Can you add a comment in the constructor about this?